docs/src/main/sphinx/release/release-406.md

```diff
@@ -57,16 +62,29 @@
   string columns stored in Parquet files. ({issue}`15269`)
 * Fix bug which could lead to more data than necessary getting read from Parquet
   files for queries with filters. ({issue}`15552`)
+* Fix query failure due to missing null counts in Parquet column indexes. ({issue}`15706`)
 
 ## Iceberg connector
 
+* Add the ID of the Trino query which led to the creation of a new Iceberg
```

Comment:
Kinda awkward .. but cant think of better/shorter wording atm

Reply:
Yeah, I spent like 10 minutes on this, and couldn't get it shorter without losing precision.

Reply:
This is an internal change and there's no need to mention in the release note in my opinion.

Reply:
@colebow Not sure why you left this entry. What's the user impact of this?

Reply:
@ebyhr didn't see your comment as a reply. Will remove.